framework-docs/src/docs/asciidoc/core/core-expressions.adoc

```diff
@@ -1775,6 +1775,11 @@ example shows how to use the Elvis operator in a `@Value` expression:
 This will inject a system property `pop3.port` if it is defined or 25 if not.
 =====
 
+CAUTION: Beware that the behavior of the Elvis operator follows boolean semantics as opposed
+to object reference semantics. Thus, rather than returning the first operand if it is non-null,
+it returns the first operand if it evaluates to true based on the rules that comprise
+https://groovy-lang.org/semantics.html#the-groovy-truth[The Groovy Truth].
+
```

Comment:
That's not entirely true.

SpEL does not honor all of Groovy's "truth" rules.

The SpEL Elvis operator checks for values that are non-null **and** non-empty (for Strings).